src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSPoolSpecification.cs

```diff
@@ -328,15 +367,27 @@ public PSStartTask StartTask
             }
         }
         
-        public System.Int32? TargetDedicated
+        public System.Int32? TargetDedicatedComputeNodes
```

Comment:
same here

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.